go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/gogo/googleapis v1.3.1 // indirect
 	github.com/gogo/status v1.1.0 // indirect
 	github.com/golang/mock v1.1.1
+	github.com/golang/protobuf v1.2.0
```

Comment:
I think we can revert the changes in `go.mod` and `go.sum`?

Reply:
reverted.